### PR TITLE
fix(curriculum): refactor step 12 catPhotoApp

### LINKED ISSUE #54767: Refactor step 12 of HTML cat photo app project
### Describe the Issue

We are still getting a ton of questions about how to do step 12 of the cat photo app project.
https://forum.freecodecamp.org/search?q=step%2012%20cat%20photo%20app%20%23html-css

So I am not entirely sure how to best update this.
Maybe there needs to be a different code example?
Maybe the hints need to be updated?

Maybe something else?

There are too many people lost on this step so I should be updated to make it clearer on what needs to be done.
So I am opening this up for discussion

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-12

### Your code

see explanation above

### Expected behavior

see explanation above

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Supravisor**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #54767

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ddbd81294d8ddc1510a8e56.md
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ddbd81294d8ddc1510a8e56.md:41]: Removing all of the text here is breaking the tests entirely.

Each step needs both a description and at least one hint.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ddbd81294d8ddc1510a8e56.md
Comment (Supravisor) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ddbd81294d8ddc1510a8e56.md:41]: Hi.  I've just started using the challenge editor.
Am going through several steps per editing session.
Currently this is still a draft, will mark as ready when all changes are finalised.

**Comment by Supravisor** [comment]
An error message says:
>Challenge slugs must be unique, but /learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-11 already exists.

This is the first time using the challenge editor, not sure how to go about resolving this.

**Comment by gikf** [comment]
> An error message says:
> 
> > Challenge slugs must be unique, but /learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-11 already exists.
> 
> This is the first time using the challenge editor, not sure how to go about resolving this.

That's caused by the adding new step itself, other languages need to be synced now: https://contribute.freecodecamp.org/how-to-work-on-practice-projects/#sync-i18n

**Comment by Supravisor** [comment]
I ran the sync-i18n script and committed the changes.
Is there anything else I need to do?

**Comment by ilenia-magoni** [comment]
Please wait for approval before running the sync, it makes review impossible.

Please remove the i18n files, then open your PR for review

File: curriculum/challenges/arabic/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc174fcf86c76b9248c6eb2.md
Comment (ilenia-magoni) [inline curriculum/challenges/arabic/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc174fcf86c76b9248c6eb2.md:1]: You have deleted the files, not removed them from the PR

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
Please change the PR to exclude i18n files, restore them as they were, do not delete them

**Comment by ilenia-magoni** [comment]
you should try with reverting https://github.com/freeCodeCamp/freeCodeCamp/pull/54983/commits/7659698d06f35c9f39061293bcdb01dc2f602757 instead

**Pull request closed without merging**
